mso/resource_mso_schema_template_anp_epg_subnet.go

```diff
@@ -283,6 +335,21 @@ func resourceMSOSchemaTemplateAnpEpgSubnetUpdate(d *schema.ResourceData, m inter
 		shared = tempVar.(bool)
 	}
 
+	primary := false
```

Comment:
Why is this explicitly assigned false and not defined as bool?

Reply:
wanted to keep consistent with shared attribute. should not make a difference in behaviour because define a var as bool is also equal to false